webapp/src/main/java/com/box/l10n/mojito/entity/Locale.java

```diff
@@ -21,6 +22,7 @@
                 @Index(name = "UK__LOCALE__BCP47_TAG", columnList = "bcp47_tag", unique = true)
         }
 )
+@BatchSize(size = 2000)
```

Comment:
based on our discussion could you had a comment to say it should be the size of the table and set to 2000 for allowing growth without having to update it?